shiny/session/_session.py

```diff
@@ -516,6 +516,8 @@ async def wrap_content_sync() -> AsyncIterable[bytes]:
 
                             wrapped_contents = wrap_content_sync()
 
+                        headers["Transfer-Encoding"] = "chunked"
```

Comment:
Maybe a comment here due to this being non-obvious enough that it didn't make it into the first edition of the code?